site/pages/view-all-disruptions.page.tsx

```diff
@@ -374,6 +388,10 @@ const ViewAllDisruptions = ({
     const [searchText, setSearchText] = useState("");
     const [servicesList, setServicesList] = useState<Service[]>([]);
     const [operatorsList, setOperatorsList] = useState<Operator[]>([]);
+    const [popUpState, setPopUpState] = useState(false);
+    const [downloadPdf, setDownloadPdf] = useState(false);
+    const [downloadExcel, setDownloadExcel] = useState(false);
```

Comment:
you could combine all these states into one useState that holds an object like {pdf: true, excel: false, csv: false}

Reply:
As there was no similarities in the implementation it would be better to use 3 states. This way the code does not need to perform additional two checks to know which document to be created. 

Could you please share the benefit of combining into a one state?

Reply:
less lines of code